docs/topics/db/models.txt

```diff
@@ -1417,6 +1536,76 @@ different database tables).
 Django will raise a :exc:`~django.core.exceptions.FieldError` if you override
 any model field in any ancestor model.
 
+
+.. _proxy-models-permissions-topic:
+
+Permissions
+-----------
+
+The proxy models work exactly the same way as concrete models and permissions
+are automatically created using their own content type. Also, proxy models
+don't inherit the permissions of the concrete model they subclass::
+
+    class Person(models.Model):
+        class Meta:
+            permissions = (('can_eat_pizzas', 'Can eat pizzas'),)
+
+    class Student(Person):
+        class Meta:
+            proxy = True
+            permissions = (('can_deliver_pizzas', 'Can deliver pizzas'),)
+
+    >>> content_type = ContentType.objects.get_for_model(
+    ...     Student,
+    ...     for_concrete_model=False,
+    ... )
+    >>> Permission.objects.filter(content_type=content_type).values_list(
+    <QuerySet [('student', 'add_student'), ('student', 'change_student'),
+    ('student', 'delete_student'), ('student', 'view_student'),
+    ('student', 'can_deliver_pizzas')]>
+
+    >>> for permission in Permission.objects.filter(content_type=content_type):
+    ...     user.user_permissions.add(permission)
+
+    >>> user.has_perm('foobar.add_person')
+    False
+    >>> user.has_perm('foobar.can_eat_pizzas')
+    False
+    >>> user.has_perms('foobar.add_student', 'foobar.can_deliver_pizzas')
+    True
+
+ContentType
+-----------
```

Comment:
This section isn't quite right yet. 

The points you're making: 

* Concrete and proxy models compare equal.
* Content type for concrete model is fetched by default. 
* `for_concrete_model=False` gives you the distinct content type. 
* Permissions are tied to the content type. 

Each perhaps needs a sentence.

Reply:
I tried to make the examples self explanatory but even if this is clear to me I can see how someone landing on the page might need more info to make sense out of it.